tests/plugins/forms/radio.js

```diff
@@ -28,17 +24,23 @@ bender.test( {
 	},
 
 	'test empty fields': function() {
-		// That feature is generally broken in IEs.
-		if ( CKEDITOR.env.ie )
-			assert.ignore();
-
+		// IE is quite problematic. You cannot remove `value` from the input. There are some hacks design specially for IE to fix `value` problem.
+		// Those customization should be removed if #844 will be fixed.
 		var bot = this.editorBot;
 
-		bot.setHtmlWithSelection( '[<input checked="checked" name="name" required="required" type="radio" value="value" />]' );
+		if ( CKEDITOR.env.ie ) {
+			bot.setHtmlWithSelection( '[<input checked="checked" name="name" required="required" type="radio" />]' );
```

Comment:
Why IE got different markup, without `[value]`? Can't it got the same markup and then just overwrite the attribute?